index.html

```diff
@@ -21,22 +21,103 @@
     <meta property="og:description" content="We're developers taking visual notes at conferences and publishing them online. We may sketchnote at your event!" />
   </head>
   <body>
-    <img src="sketchnoteclub-logo.png" alt="Sketchnote Club Logo showing a pen between a less-than and greater-then sign.">
-    <h1>Sketchnote Club</h1>
-    <p>We may sketchnote at your conference!</p>
-    <p>Get in touch:</p>
-    <a href="mailto:sketchnoteclub@gmail.com">sketchnoteclub@gmail.com</a>
-
-    <h3>Follow us:</h3>
-    <p><a href="https://twitter.com/sketchnoteclub">Twitter</a></p>
-    <p><a href="https://www.instagram.com/sketchnoteclub/">Instagram</a></p>
-
-    <div id="coming-soon">
-      <strong>Coming soon:</strong>
-      <p>What is Sketchnoting?</p>
-      <p>Who are we?</p>
-      <p>What do we offer?</p>
-      <p>What's the price?</p>
+    <div class="page">
+
+      <div class="container header">
+        <img src="sketchnoteclub-logo.png" alt="Sketchnote Club Logo showing a pen between a less-than and greater-then sign.">
+        <h1>Sketchnote Club</h1>
+        <p>We'll sketchnote your conference!</p>
+      </div>
+
+      <div class="container">
+        <p>We are a group of folks who get really excited about learning and teaching. Sketchnotes is an awesome way to absorb & digest information and share it with the community.</p>
+      </div>
+
+
+      <div class="container contributors">
+        <h3>collaborative instigators</h3>
```

Comment:
`instigators` is probably super cool slang (for native english speakers) but I am afraid in the German market no body will understand it. 
Could we change it to "sketchnoters" or "members" or something more telling? 
Sketchnoters may also be difficult as many people have not heard of it ever.

Reply:
Ah yes, this was me trying to be cute (also making this late at night). How about just "collaborators"? I like the idea that we're all building this together (:

Reply:
Yes, "collaborators" is cool 👍